test/test_build.py

```diff
@@ -39,6 +39,43 @@ class ImageBuildResult(NamedTuple):
     metadata: dict = {}
 
 
+def get_distribution_and_version(test_vm, build_ansible_container, username, password):
+    """Run Ansible playbook to get distribution and version"""
+
+    output = test_vm.execute_ansible(build_ansible_container,
+                                     "./test/ansible-playbooks/get_distro_version.yml",
+                                     username,
+                                     password)
+
+    print(output)
```

Comment:
is this a debug leftover? If not, maybe worth adding some context to the print, e.g. `print(f"output from ansible get_distro_version playboot {output}")` or something